docs/pages/components/dialog/examples/ExPromptDialog.vue

```diff
@@ -29,47 +39,71 @@ export default {
                 message: `What's your name?`,
                 inputAttrs: {
-                    placeholder: 'e.g. Walter',
+                    placeholder: "e.g. Walter",
                     maxlength: 10
                 },
                 trapFocus: true,
-                onConfirm: (value) => this.$buefy.toast.open(`Your name is: ${value}`)
-            })
+                onConfirm: value =>
+                    this.$buefy.toast.open(`Your name is: ${value}`)
+            });
         },
         promptNumber() {
             this.$buefy.dialog.prompt({
                 message: `What's your age?`,
                 inputAttrs: {
-                    type: 'number',
-                    placeholder: 'Type your age',
-                    value: '18',
+                    type: "number",
+                    placeholder: "Type your age",
+                    value: "18",
                     min: 18,
                     max: 99
                 },
                 trapFocus: true,
-                onConfirm: (value) => this.$buefy.toast.open(`Your age is: ${value}`)
-            })
+                onConfirm: value =>
+                    this.$buefy.toast.open(`Your age is: ${value}`)
+            });
         },
         promptNotClosed() {
             this.$buefy.dialog.prompt({
-                message: 'Send your message to moon 🚀',
+                message: "Send your message to moon 🚀",
+                inputAttrs: {
+                    type: "text",
+                    placeholder: "My message is...",
+                    value: "Hello moon!"
+                },
+                confirmText: "Send",
+                trapFocus: true,
+                closeOnConfirm: false,
+                onConfirm: (value, { close, startLoading }) => {
+                    startLoading();
+                    this.$buefy.toast.open(`Your message is sending...`);
+                    setTimeout(() => {
+                        this.$buefy.toast.open(`Success message send!`);
+                        close();
+                    }, 2000);
+                }
+            });
+        },
+        promptNotClosedWithLoading() {
+            this.$buefy.dialog.prompt({
+                message: "Send your message to moon 🚀",
                 inputAttrs: {
-                    type: 'text',
-                    placeholder: 'My message is...',
-                    value: 'Hello moon!'
+                    type: "text",
+                    placeholder: "My message is...",
+                    value: "Hello moon!"
                 },
-                confirmText: 'Send',
+                confirmText: "Send",
                 trapFocus: true,
                 closeOnConfirm: false,
-                onConfirm: (value, {close}) => {
-                    this.$buefy.toast.open(`Your message is sending...`)
+                onConfirm: (value, { startLoading, cancelLoading }) => {
+                    startLoading();
```

Comment:
@zhanghengxin Sorry for confusion, but I meant to remove this line from the library by the following comment:
> Could you revert to your initial idea of calling startLoading explicitly?

Reply:
What's meaning？
What should I do?

Reply:
Please remove the line 98: `startLoading();`

Reply:
If there is no startLoading(), then there is no Loading

Reply:
Sorry, I was wrong.

Reply:
hah，I'm glad to see even experts make mistakes sometimes. It makes me feel better about my own mistakes.

Reply:
> Sorry, I was wrong. I was looking at a wrong file.

I meant: https://github.com/buefy/buefy/pull/3897/files#r1356019615